src/utils/trust/individual.trustee.mapper.ts

```diff
@@ -33,6 +33,10 @@ export const mapIndividualTrusteeToSession = (
     is_service_address_same_as_usual_residential_address: (formData.is_service_address_same_as_usual_residential_address) ? Number(formData.is_service_address_same_as_usual_residential_address) : 0,
     sa_address_care_of: '',
     sa_address_po_box: '',
+    is_individual_still_involved_in_trust: formData.stillInvolved,
```

Comment:
Are we translating 0s and 1s to "Yes" and "No"?

Reply:
I've followed the example used on the Trust still involved question - https://github.com/companieshouse/overseas-entities-web/blob/main/src/model/trust.model.ts#L36
The template will convert the 0's and 1's to yes and no.

Reply:
actually i can see what you mean, like this - https://github.com/companieshouse/overseas-entities-web/blob/main/src/utils/trust/details.mapper.ts#L75
I'll update it.